knimin/tests/test_ag_edit_barcode.py

```diff
@@ -0,0 +1,108 @@
+from unittest import main
+
+from tornado.escape import url_escape
+
+from knimin.tests.tornado_test_base import TestHandlerBase
+from knimin import db
+
+
+class TestAGEditBarcodeHandler(TestHandlerBase):
+    def test_get_not_authed(self):
+        response = self.get('/ag_edit_barcode/')
+        self.assertEqual(response.code, 200)
```

Comment:
documentation of tornado specifies that a 200 is returned for an unauthed user for a get method, but a 403 for an unauthed user for a post method. Thus, it is working as specified - but maybe not as expected. But I guess we should not strive to change this basic behaviour of tornado.